integrations/gen_doc_collector_page.py

```diff
@@ -0,0 +1,75 @@
+import json
+
+"""
+This script reads the integrations/integrations.js file and generates the list of data collection integrations inside collectors/COLLECTORS.md, with proper links that Learn can replace into Learn links.
+"""
```

Comment:
```suggestion
"""
This script reads the integrations/integrations.js file and generates the list of data collection integrations inside collectors/COLLECTORS.md, with proper links that Learn can replace into Learn links.
"""

import json
```

Imports should generally go after the module docstring.